diff --git a/BUILD.bazel b/BUILD.bazel
index 415ef81df9cc..126f11d9d267 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -35,6 +35,7 @@ _aarch64_additional_kmi_symbol_lists = [
     "android/abi_gki_aarch64_virtual_device",
     "android/abi_gki_aarch64_virtual_device_removed",
     "android/abi_gki_aarch64_vivo",
+    "android/abi_gki_aarch64_pstore_screen",
     "android/abi_gki_aarch64_xiaomi",
 ]
 
diff --git a/android/abi_gki_aarch64_pstore_screen b/android/abi_gki_aarch64_pstore_screen
new file mode 100644
index 000000000000..dae528ce1882
--- /dev/null
+++ b/android/abi_gki_aarch64_pstore_screen
@@ -0,0 +1,7 @@
+[abi_symbol_list]
+  pstore_screen_display_register
+  pstore_screen_display_unregister
+  pstore_screen_drm_sidecar_register
+  pstore_screen_drm_sidecar_unregister
+  pstore_screen_panic_start
+  pstore_screen_panic_wait
diff --git a/build.config.gki.aarch64 b/build.config.gki.aarch64
index 263645e519ac..6d2778c05bdb 100755
--- a/build.config.gki.aarch64
+++ b/build.config.gki.aarch64
@@ -48,6 +48,7 @@ TRIM_NONLISTED_KMI=${TRIM_NONLISTED_KMI:-1}
 KMI_SYMBOL_LIST_STRICT_MODE=${KMI_SYMBOL_LIST_STRICT_MODE:-1}
 KMI_ENFORCED=1
 
+android/abi_gki_aarch64_pstore_screen
 BUILD_SYSTEM_DLKM=1
 MODULES_LIST=${ROOT_DIR}/${KERNEL_DIR}/android/gki_system_dlkm_modules
 MODULES_ORDER=android/gki_aarch64_modules
